content/en/cloud-computing.md

```diff
@@ -5,7 +5,7 @@ category: concept
 tags: ["infrastructure", "fundamental", ""]
 ---
 
-Cloud computing offers CPU power, storage, and network capabilities over the internet, 
+Cloud computing offers CPU power, storage, and network capabilities over the network, 
```

Comment:
Maybe something like this might help clarify?

```suggestion
Cloud computing offers CPU power, storage, and network capabilities over the a network (or even the internet), 
```

Reply:
@scaarup I don't know, but I find `...network capabilities over the network` sounds super strange and confusing.  While the original sentence might not be profound enough it is easier to understand. Maybe we do something completely different. 
```suggestion
Cloud computing offers CPU power, storage, and network capabilities, 
```

Reply:
Yeah I like that 👌🏻